packages/shared-http-pieces/src/lib/information.ts

```diff
@@ -0,0 +1,27 @@
+let _repository = 'https://github.com/skyra-project';
+let _invite = '';
+
+export function setRepository(repository: string) {
+	_repository = repository.includes('/') ? repository : `https://github.com/skyra-project/${repository}`;
```

Comment:
```suggestion
	_repository = repository.startsWith('http') && repository.includes('/') ? repository : `https://github.com/skyra-project/${repository}`;
```
Feels safer

Or better yet, do a try parse with `new URL()`